### ISSUE TITLE
[Feature Request] - Federated Auth in an Example App

### ISSUE THREAD

**Issue by iandanforth** [supnate/rekit#109]
Many apps will require their users to be authenticated. An example of how to add auth (e.g. google auth) to a rekit app would be an extremely useful addition to the example application.

**Comment by supnate** [comment]
Thanks @iandanforth for the suggestion! Since now Rekit creates apps by create-react-app, many tutorial should apply to Rekit too, haha! I just found one https://medium.appbase.io/how-to-implement-authentication-for-your-react-app-cf09eef3bb0b or https://auth0.com/blog/reactjs-authentication-tutorial/ .

However, I agree providing an out of box sample will be much helpful. I definitely will add more when have time. Thanks!

**Comment by jihadismail8** [comment]
yes please , i been waiting for a new tutorial where  alot of things will be used . like to install material uo , Auth , create-react-app ,and router . we are waiting for such a tutorial . please make it

**Comment by chiester** [comment]
Are there general recommendations as to how to integrate an external auth solution in the meantime? What would the best practice be? For instance, if I'm using Okta's SDK and <SecureRoute> which is wrapped by react-router's <Route> would that go into App.js? Or should it go into Root.js?

**Comment by chiester** [comment]
FYI, if anyone is asking the same question I asked above, you can find a pretty good approach at https://github.com/supnate/rekit/issues/115
